src/picongpu/include/particles/ionization/byCollision/ThomasFermi/ThomasFermi_Impl.hpp

```diff
@@ -75,7 +75,11 @@ namespace ionization
 
         using FrameType =  typename SrcSpecies::FrameType;
 
-        /* specify field to particle interpolation scheme */
+        /** specify field to particle interpolation scheme
+         *
+         * @todo this needs to be done independently/twice if ion species (rho) and electron
+         *       species (ene) are of different shape
```

Comment:
Sure sure, that's a follow-up feature and I have a hunch that it requires some careful thinking. :laughing: